wiki/en/Running-a-Server.md

```diff
@@ -136,63 +116,62 @@ Note also that mean ADSL2 transfer rate is 10 Mbit/s for downstream and 1 Mbit/s
+# Starting a Server
 
 
 * **Windows users** - Use the "Jamulus Server" icon in the Windows Start menu.
 
 * **macOS users** - Double-click the "Jamulus Server" icon in Applications (assuming you put the files from the install there as per [these instructions](Installation-for-Macintosh)).
 
-* **Linux users** - Open a terminal window (`CTRL+ALT+t` on Ubuntu and related distros) and type `jamulus -s`. Hit return and you should see the server control window. You can stop the server by closing the server window, or by using `CTRL+C` in the terminal. _(To run a headless server [read this guide](Server-Linux))_
+* **Linux users** - Open a terminal window (`CTRL+ALT+t` on Ubuntu and related distros) and type `jamulus -s`. Hit return and you should see the Server control window. You can stop the Server by closing the Server window, or by using `CTRL+C` in the terminal. _(To run a headless Server [read this guide](Server-Linux))_
 
 
 ## Server Setup
 
-<figure><img src="{% include img/en-screenshots/server-window-setup.inc %}" style="width:80%; border:5px solid grey;" loading="lazy" alt="Image of the Jamulus server setup window"></figure>
-
-### Make My Server Public
+<figure><img src="{% include img/en-screenshots/server-window-setup.inc %}" style="width:80%; border:5px solid grey;" loading="lazy" alt="Image of the Jamulus Server setup window"></figure>
 
-By default, you will be running a Private Server and need to [read these instructions](#running-a-private-server) to have others connect to you in this mode.
+### The Directory list 
 
-When making your server public, you should see a confirmation message saying whether your server has registered successfully. If not, and you leave your server running, it will keep trying to register until a free slot becomes available.
+**None**: By default, you will not be connected to a Directory and will be running in unregistered mode. [Read these instructions](#running-an-unregistered-server) to have other people connect to your Server in this mode.
 
+**Genre**: To allow other people to see your Server on one of the built-in Public Directories, select your desired genre Directory. You should see a confirmation message saying whether your Server has registered successfully. If not, and you leave your Server running, it will keep trying to register until a free slot becomes available.  
 
-### Genre
-
-Because there is a limit of 150 servers per Directory, you need to select which Directory Server you want to advertise your server on. Select a genre if you intend to limit players to that style (players can then see your server in their genre list). This will make your server appear in the relevant server list for clients.
-
+**Custom**: This sets your Server to be a Directory on which other Servers can be listed. Note that when setting this mode, you may want also to specify the location of Server list file (See "Options" below) unless you want to disconnect all listed Servers when restarting Jamulus.
```

Comment:
Still "No!"

The main reason for using "Custom" is to connect to directories other than the built in ones.

Reply:
I'm confused. Can you suggest some wording for this?

Reply:
Like this:

----

**Custom**: This allows you to specify a custom directory on which to be listed.  See the "Options" tab for the Custom Directory server address you want to use.

To run your Server _as_ a Directory, you need to set the Custom Directory server address as "localhost" or "127.0.0.1" and set the "Genre" to "Custom".

----

Reply:
Side node: Why are we still referring to "Genre" not "List" in the server? I think the client is saying "List"